extensions/amp-soundcloud/0.1/test/validator-amp-soundcloud.out

```diff
@@ -46,11 +46,14 @@ amp-soundcloud/0.1/test/validator-amp-soundcloud.html:39:2 The attribute 'data-v
 >>   ^~~~~~~~~
 amp-soundcloud/0.1/test/validator-amp-soundcloud.html:42:2 The attribute 'data-trackid' in tag 'amp-soundcloud' is set to the invalid value 'mahler_number_6.ogg'. (see https://amp.dev/documentation/components/amp-soundcloud) [DISALLOWED_HTML]
 |                    layout="fixed-height"></amp-soundcloud>
-|    <!-- invalid: bad layout -->
+|    <!-- valid: responsive layout -->
 |    <amp-soundcloud height=166 width=42 data-trackid="243169232" data-visual="true"
->>   ^~~~~~~~~
-amp-soundcloud/0.1/test/validator-amp-soundcloud.html:45:2 The specified layout 'RESPONSIVE' is not supported by tag 'amp-soundcloud'. (see https://amp.dev/documentation/components/amp-soundcloud) [AMP_LAYOUT_PROBLEM]
 |                    layout="responsive"></amp-soundcloud>
+|    <!-- valid: intrinsic layout -->
```

Comment:
Hmm, it looks like intrinsic is missing from the validator layouts,

Reply:
Good catch! Although most video/audio components are lacking intrinsic too...